piccolo/apps/asgi/commands/templates/app/requirements.txt.jinja

```diff
@@ -2,7 +2,4 @@
 {{ router_dependency }}
 {% endfor -%}
 {{ server }}
-jinja2
-piccolo[postgres]
-piccolo_admin
-requests
+piccolo[postgres]>=1.0.0
```

Comment:
Not sure if this will cause any problems ... i.e. it means that all of routers must support Pydantic v2. Will see when the CI runs.